tag_generator/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "tag_generator"
```

Comment:
Some alternative names:

- `tag-generator` because you use hyphen in your other crate names
- `taggen` because is looks and sounds funny

Reply:
Good catch. I vote for `tag-generator`.